services/wallet/activity/activity_test.go

```diff
@@ -202,7 +203,7 @@ func TestGetActivityEntriesAll(t *testing.T) {
 	require.Equal(t, 4, len(entries))
 
 	// Ensure we have the correct order
-	var curTimestamp int64 = 4
+	var curTimestamp int64 = 5
```

Comment:
This looks confusing to me. I'm not sure I understand why do we now start having timestamps from 2. Could you please clarify the source for this index-timestamp mismatch?